source/common/access_log/access_log_formatter.cc

```diff
@@ -245,14 +244,16 @@ std::vector<FormatterProviderPtr> AccessLogFormatParser::parse(const std::string
       pos += 1;
       int command_end_position = pos + token.length();
 
-      if (absl::StartsWith(token, "REQ(")) {
+      const bool req_without_query = absl::StartsWith(token, "REQ_WITHOUT_QUERY(");
```

Comment:
hmm looking at this code would it make more sense to have `REQ_WITHOUT_QUERY` just take the header key as an input instead of a string to transform? right now it has to be used with REQ so it doesn't seem like its providing any additional flexibility